samza-api/src/main/java/org/apache/samza/checkpoint/CheckpointV1.java

```diff
@@ -44,7 +44,7 @@ public CheckpointV1(Map<SystemStreamPartition, String> offsets) {
     this.offsets = offsets;
   }
 
-  public short getVersion() {
+  @Override public short getVersion() {
```

Comment:
Looks like the style in the other parts of code are to have `@Override` on a new line above the method. i.e.:
```
@Override
public short getVersion() {
```
Sorry to request this, but could you please update that in each of the files?

Reply:
Fixed in https://github.com/apache/samza/pull/1539